platformio.ini

```diff
@@ -10,13 +10,21 @@
 [platformio]
 include_dir = controller/include
 src_dir = controller/src
+test_dir = controller/test
+; Make `platformio run` build just env:uno.  We skip env:native for now because
+; it doesn't build!
+default_envs = uno
 
+[env]
+lib_ldf_mode = deep+
+lib_extra_dirs = common/libs/
+build_flags = -Icommon/include/
 
 [env:uno]
 platform = atmelavr
 board = uno
```

Comment:
Current target is a nano, PlatformIO has a board for that. Though I guess that could change soon.

Reply:
I am hesitant to touch this because I don't want to mess up anyone who *is* currently flashing their board, and I don't have a board to test anything on.

Unless you're confident it'll be OK?

Reply:
Good point...I'm sure someone is/will try using that case. Maybe add a new env then for nano?